ouroboros-network/src/Ouroboros/Network/Block.hs

```diff
@@ -50,15 +51,11 @@ newtype SlotNo = SlotNo { unSlotNo :: Word64 }
 newtype BlockNo = BlockNo { unBlockNo :: Word64 }
   deriving (Show, Eq, Ord, Enum, Bounded, Num, Serialise, Generic)
 
+-- | Header hash
+type family HeaderHash b :: *
```

Comment:
Let's check if this is still needed or not.

Reply:
It is. I can (and need to) give a `HeaderHash` definition for `SimpleBlock' c ext ext'`, but can give a `HasHeader` instance only for `SimpleBlock c ext = SimpleBlock' c ext ext`.